docs/source/support/release-notes.rst

```diff
@@ -12,6 +12,15 @@ HEAD — ongoing
 DAML
 ~~~~
 
+- **BREAKING CHANGE - DAML Standard Library**: Moved the ``Tuple`` and ``Either`` types to ``daml-prim:DA.Types``
+  rather than exposing internal locations.
+
+How to migrate:
```

Comment:
```suggestion
  How to migrate:
```
Adding the leading spaces makes the "migration notes" look more as part of the bullet point it belongs to.

Reply:
I was copying from the Ledger API section a few lines below which has made this mistake. Should I fix that too?

Reply:
Yeah, that was my mistake. Sorry. I'm fixing it in #1107.